src/control/server/server.go

```diff
@@ -145,6 +155,8 @@ func (srv *server) shutdown() {
 // initNetwork resolves local address and starts TCP listener then calls
 // netInit to process network configuration.
 func (srv *server) initNetwork(ctx context.Context) error {
+	defer srv.duration(track("time to init network"))
```

Comment:
This is an interesting subtlety I wasn't aware of, that the parameters are evaluated here, before the deferred function executes.